rotkehlchen/db/upgrades/v35_v36.py

```diff
@@ -542,6 +544,31 @@ def _add_okx(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_okx')
 
 
+def _upgrade_liquity_staking_events(write_cursor: 'DBCursor', conn: 'DBConnection') -> None:
+    log.debug('Enter _upgrade_liquity_staking_events')
+    with conn.read_ctx() as read_cursor:
+        read_cursor.execute(
+            'SELECT identifier, extra_data FROM history_events WHERE type=? AND subtype IN (?, ?) AND counterparty=?',  # noqa: E501
+            (
+                HistoryEventType.STAKING.serialize(),
+                HistoryEventSubType.DEPOSIT_ASSET.serialize(),
+                HistoryEventSubType.REMOVE_ASSET.serialize(),
+                CPT_LIQUITY,
+            ),
+        )
+        for (identifier, serialized_extra_data) in read_cursor:
+            if serialized_extra_data is None:
+                continue
+            extra_data = json.loads(serialized_extra_data)
+            upgraded_extra_data = {'liquity-staking': extra_data}
```

Comment:
if the `liquity-staking` and `sub-swaps` is something this deep it should be a constant somewhere